README.md

```diff
@@ -9,3 +9,4 @@ This repository holds various examples that showcases Screenly's features and AP
 * [JavaScript Injectors](https://github.com/Screenly/playground/tree/master/javascript-injectors/)
 * [Weather App](https://github.com/Screenly/weather-app)
 * [Clock App](https://github.com/Screenly/clock-app)
+* [API](./api/README.md)
```

Comment:
it's not an API example. It's really a demo of the JavaScript injector, so it probably rather belongs as part of that.

Reply:
Actually, I think it is an API example more than JS injection example - JS injection is just a convenient web asset - we could use any other asset type in this example.